app/views/member_mailer/registration_confirmation.html.erb

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+    <style type="text/css">
+      body#welcome h1, body#meeting h1 {
```

Comment:
At the time I wrote this, I couldn't figure out how to get styles to work in an email without inlining it [note](https://github.com/ilyakava/sup/blob/master/app/assets/stylesheets/emails.scss#L1), would be great if someone fixed this in all the emails, maybe starting with this one (if its possible, I'm not attached to `premailer`, but things might have gotten easier in the 1.5 years that have elapsed